modules/apps/headless/headless-asset-library/headless-asset-library-impl/rest-openapi.yaml

```diff
@@ -4,6 +4,12 @@ components:
             description:
                 Represents an Asset Library
             properties:
+                assetLibraryId:
+                    description:
+                        The asset library's ID.
+                    format: int64
+                    readOnly: true
+                    type: integer
```

Comment:
If this is the main id, shouldn't this be named `id` for consistency with other entities?

Reply:
we can rename 
assetLibraryId > id
id > siteId


what do you think about it?


note : BUT on the methods should be named assetLibraryId/ siteId FYI